commands/pull.js

```diff
@@ -22,6 +22,14 @@ function parseURL (db) {
   return db
 }
 
+function parseExclusions (rawExcludeList) {
+  return (rawExcludeList || '').split(';').map(function(tname) {
+      return tname.trim();
+    }).filter(function(tname) {
+      return (tname != '');
+    });
+}
```

Comment:
So, this is going to preclude *some* esoteric table names, but given the CLI doesn't support repeated flags (it doesn't, right @jdxcode @RasPhilCo?), that's going to happen one way or another, and most commonly-used table names are going to be supported by this.

Reply:
You're right, CLI does not support repeated flags (at least that is what I could deduce from the other commands code AND documentation). So I've just followed the way it was done for other commands : same technique & separator :)